### PR TITLE
Create 00824.js

### PR DESCRIPTION
**Pull request by Toriiaa**

### CODE REVIEW

**Comment by Toriiaa** [comment]
please notify me if it is accepted.

**Review by onedebos — COMMENTED** [review]
Hi. Why do you have an HTML doc in your JS file? Your js output should be in the form` console.log("Hello World, this is Itunuloluwa Fatoki with HNGi7 ID HNG-01234 using javascript for stage 2 task itunuworks@gmail.com)` Also, your filename should be in the form `01234-itunuworks.js `or `hng-83932-racheal.py`

**Review by Toriiaa — COMMENTED** [review]
is this better.

**Review by onedebos — COMMENTED** [review]
Much better. But you need to format your HNG-ID. Your script should be in the form `Hello World, this is Itunuloluwa Fatoki with HNGi7 ID HNG-01234 using javascript for stage 2 task itunuworks@gmail.com`. Also, please rename your script name to the form `01234-itunuworks.js` or `hng-83932-racheal.py`

**Review by onedebos — CHANGES_REQUESTED** [review]
I can see you've updated your filename. Well done! One last thing, fix the HNG-ID to use the format HNG-03232

**Review by Toriiaa — COMMENTED** [review]
thank you . i made all the corrections. Approved?

**Comment by onedebos** [comment]
Nearly there! Please remove "i am using". You should only have "using". Please make sure your text matches the format I specified above, including white-spaces.

**Review by onedebos — CHANGES_REQUESTED** [review]
Your string should output the format  - `Hello World, this is [full name] with HNGi7 ID [ID] using [language] for stage 2 task [email_address]`

**Review by Toriiaa — COMMENTED** [review]
i think i have done everything.

**Review by onedebos — CHANGES_REQUESTED** [review]
you should have **`with HNGi7 ID`** not `with HNGi7 ID` **`is`**

**Review by Mahalinoro — COMMENTED** [review]
Also, console.log(Merit) not console.log("Merit")

**Review by Toriiaa — COMMENTED** [review]
DONE

File: scripts/00824-meritoriaghe.js [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
       let Merit;
       Merit = `Hello World, this is ${myName} with HNGi7 ID is ${myId} using ${myLanguage} for stage2 task ${myEmail}`;
```
Comment (onedebos) [inline scripts/00824-meritoriaghe.js]: you still have **is** here.

**Review by onedebos — CHANGES_REQUESTED** [review]
see my comment below.

**Review by Toriiaa — COMMENTED** [review]
DONE

**Review by onedebos — APPROVED** [review]

**Pull request merged**